docs/messageData.md

```diff
@@ -1025,3 +1030,496 @@ the organizer forgets to scan an attendee’s public key.
+            "description": "[Timestamp] time created in UTC",
+            "type": "integer",
+            "minimum": 0
+        },
+        "start_time": {
+            "description": "[Timestamp] start time of the election in UTC",
+            "type": "integer",
+            "minimum": 0
+        },
+        "end_time": {
+            "description": "[Timestamp] end time of the election in UTC",
+            "type": "integer",
+            "minimum": 0
+        },
+        "questions": {
+            "description": "Array of questions",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "contentEncoding": "base64",
+                        "$comment": "Question ID: Hash : HashLen('Question', election_id, question)"
+                    },
+                    "question": {
+                        "type": "string",
+                        "$comment": "voting topic"
+                    },
+                    "voting_method": {
+                        "type": "string",
+                        "enum": ["Plurality", "Approval"],
+                        "$comment": "supported voting method"
+                    },
+                    "ballot_options": {
+                        "description": "[Array[String]] ballot options",
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "$comment": "ballot option"
+                        },
+                        "minItems": 2,
+                        "uniqueItems": true
+                    },
+                    "write_in": {
+                        "type": "boolean",
+                        "$comment": "whether write-in is allowed"
+                    }
+                },
+                "additionalProperties": false,
+                "required": [
+                    "id",
+                    "question",
+                    "voting_method",
+                    "ballot_options",
+                    "write_in"
+                ]
+            },
+            "minItems": 1,
+            "uniqueItems": true
+        }
+    },
+    "additionalProperties": false,
+    "required": [
+        "object",
+        "action",
+        "id",
+        "lao",
+        "name",
+        "version",
+        "created_at",
+        "start_time",
+        "end_time",
+        "questions"
+    ]
+}
+
+```
+
+## Casting a vote (election#cast_vote)
+
+🧭 **RPC Message** > **RPC payload** (*Query*) > **Query payload** (*Publish*) >
+**Mid Level** > **High level** (*election#cast_vote*)
+
+A member of the LAO can cast a vote by publishing an election/cast_vote message to the
+election’s channel. Each member may cast multiple votes, only the last one will be counted.
+If write-in is allowed for the election then the vote has to have a write-in.
```

Comment:
Sentence may be worth clarifying:
Is it that the write-in is mandatory for the election, and so the field is mandatory,
or is it that the write-in is allowed for the election, and so the field is mandatory (but may be null) ?

Reply:
The cast vote either has a vote field or a write-in field, in the schema it is defined as a `oneOf`